docs/examples/queries/shop.md

```diff
@@ -0,0 +1,133 @@
+---
+id: shop
+title: Shop
+---
+
+In this example, we will query for a shop, its shop levels, and order messages by the shop slug.
+In this case, we've chosen the Summit of Time Compendium as the shop.
+
+## Example
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Request-->
+
+```graphql
+query Shop ($id: ID, $slug: String) {
+  shop (id: $id, slug: $slug) {
```

Comment:
we want to use valid queries in the examples (and real/usable if possible), so keep $slug in this query and get rid of $id since $slug is the one you used in the query variables